package.json

```diff
@@ -6,7 +6,7 @@
     "start": "node ./bin/www",
     "test": "export NODE_ENV=test && mocha test"
```

Comment:
I know this is not your fault, but there is no need to pollute the env variables by exporting NODE_ENV. you can use this instead:

```
     "test": "NODE_ENV=test && node_modules/mocha/bin/mocha test" 
```

What do you think?